app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3 and command.split()[0] == "mv":
```

Comment:
The condition checks if the command has three parts and starts with 'mv', which is correct according to the checklist.